.github/ISSUE_TEMPLATE/moderator-onboarding.md

```diff
@@ -0,0 +1,15 @@
+---
+name: "Moderator onboarding"
+about: "for people joining R-multiverse as moderators"
+labels: "moderator onboarding"
+assignees: r-multiverse/administrators
+---
+
+I would like to join the R-multiverse [moderator](https://r-multiverse.org/team.html#moderators) team.
+
+## Onboarding process
```

Comment:
split into checklist for the candidate and checklist for the admin?

Reply:
Added in 6d5f1e452c85af9d8ec1c2292712796082532bfd